### ISSUE TITLE
Unknown Recipe's for each building

### ISSUE THREAD

**Issue by smmmadden** [ldtteam/MinecoloniesWiki#746]
## Page(s)
I checked the Misc | Recipes page and it shows some of them but mostly for the items to create buildings from.  The individual building pages talks about recipes but doesn't get into any details about what is available.  For example, the Plantation has 480 recipes that could be crafted, but I have no clue what they all are.  Paper, Sugar, Books, Sticks seem to be it.  Given there is logic that accepts specific recipes, is there code that shows what they all are for each building?  This is the biggest area that I spend a lot of time on for every colony.

## Information Missing
Wasn't there also a setting in the config file to either require a recipe or not for each building? I thought I mentioned that back in 1.16.5 or earlier since it is monotonous to have to repeat the same recipe over and over (like multiple builders - adding them once would be far better IMHO).

I'd even be willing to update the wiki page if I had a list to start with so it benefits everyone like I did with the list of names. :-)
Thanks,
Steve

**Comment by Talyda** [comment]
It wouldn't be feasible to have every recipe that every building can make, because those are not a set list. Recipes from other mods, or modified by other mods changes all of that. Use JEI - if you want something made, JEI should tell you which worker will make it . If somewhere like the plantation can make 480 recipes (where did you find that number BTW?) then you have to take into account mods that add different ways to make each item. You can have 50 mods that have 50 different ways to make paper - the plantation can probably make all of those, but where would anyone ever find that list to start with?

And no, there is not, and as far as I can remember never has been a way to not teach recipes. they don't want it to work that way. Not everyone would want all their workers to know the same recipes - and for a lot of workers you absolutely do not want more than one of them knowing the recipe =)

**Comment by Talyda** [comment]
>This is what I see for the Plantation Hut. 
That just means they know 4 of potentially 480 recipes that you can teach them (which I thought should be 160, not 480)
It's a standard number for each crafter - I honestly don't know why that one says 480, unless they changed the limit at some point!

As for guessing, mostly all of the huts wiki pages tell you what they can make - plantation for example "The Planter can also craft paper, books, sugar, and anything made with bamboo. " Some of the buildings (not all, cos someone needs to go to the effort of writing it up) have a red question mark on the GUI which have some basic information about the hut, which would include that line above for the plantation (as an example) 
![image](https://user-images.githubusercontent.com/17323101/160925808-3eec8867-344e-444e-86d8-bada18da3e95.png)
 - you could ask on discord if they were planning on adding that to each building and maybe help with that?  (unless I am misunderstanding what you mean!)

**Comment by Talyda** [comment]
Oh! Thanks to Rentsuru, 160 is the default amount of recipes that they can learn, it goes up to 480 once you do all the research - my confusion!

**Comment by smmmadden** [comment]
thanks - it is the "anything" that gets me.  That puts assumption on the player they know everything that can be crafted with bamboo. LOL  I'm good after playing since 1.8.x but with all the changes coming in every release, it's hard to keep track of just the vanilla ones and now there are many huts with their own recipes (some don't work for all).   I keep forgetting to look at that page and I do see Restrictions on most hut and talks about each level that unlocks certain tools.  Is that the same as unlocking recipes for the same?  Would be great if it is to narrow the lists down more.

I was expecting that the builder (or Sawmill) could craft the following: pressure plates, doors, trapped doors, fences, gates, bowls, boats and barrels but it wouldn't accept those.  So not all vanilla recipes seem to match up with the right huts.  The list could be very simple like you and I just listed. It doesn't need graphics to figure out, JEI can do that already.

I've done a lot of documenting of features (for my 25+ years as a Software Engineer/Systems Engineer) and for some mods/plugins (which I also have several I develop).  Repeating the same steps over and over takes the fun away from actually playing the game (which my wife/I do every night on our servers).  If Colony 1 has 6 builder huts, I'd want all 6 to take advantage of the same recipes to save from doing spending hours training every single recipe 6 times, just for that colony only to have to do the same in the next colonies. Not to mention telling players how to figure this out.  LOL  That was the feedback I provided to @Raycoms in the past to consider adding the logic for at least the list for each hut and ultimately to support a config file that can save time.  I appreciate the insight and help! 
Regards,
Steve
